spark/src/main/scala/org/apache/spark/sql/delta/stats/StatsCollectionUtils.scala

```diff
@@ -27,6 +28,7 @@ import org.apache.spark.sql.delta.actions.AddFile
 import org.apache.spark.sql.delta.sources.DeltaSQLConf
 import org.apache.spark.sql.delta.stats.DeltaStatistics._
 import org.apache.spark.sql.delta.util.{DeltaFileOperations, JsonUtils}
+import org.apache.spark.sql.delta.util.threads.DeltaThreadPool
```

Comment:
The package name seems to be wrong. Are you missing a change?

Reply:
Ah yes, I am depending on this PR https://github.com/delta-io/delta/pull/2154 being merged first. I didn't want to include the changes of it in this PR, as it would make the code review harder. But once the other PR is merged, this should match up.